backend/ohq/permissions.py

```diff
@@ -504,3 +508,50 @@ def has_permission(self, request, view):
             return True
 
         return True
+
+class BookingPermission(permissions.BasePermission):
```

Comment:
seems like you're always doing occurence.event so why not make this get_membership_from_occurence? also this is better as a static function